WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -158,8 +159,8 @@ public void onErrorResponse(VolleyError volleyError) {
         WordPress.getRestClientUtilsV1_1().get(path, params, null, listener, errorListener);
     }
 
-    public static void updateRole(final String blogId, long personID, String newRole, final int localTableBlogId,
-                                  final UpdateUserCallback callback) {
+    public static void updateRole(Context context, final String blogId, long personID, Role newRole,
```

Comment:
It looks like we can drop the `context` param from `updateRole` and `sendInvitations` now.